firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/metadata/MetaDataStoreTest.java

```diff
@@ -43,6 +47,8 @@ public class MetaDataStoreTest extends CrashlyticsTestCase {
 
   private static final String ESCAPED = "\ttest\nvalue";
 
+  private static final List<RolloutAssignment> ROLLOUTS_STATE = new ArrayList<>();
```

Comment:
Does this need to be static? I am worried some unexpected behaviour could happen when running multiple tests, and state from one test affecting another. If this is changed to a member variable, then the setup will be more consistent.

Reply:
Hmmm I can change them to non-static, but I do use it as a static variable. After ROLLOUTS_STATE is configured in setup() and I do not expect ant test case change the value. It is used as expected result comparison.